tests/MongoDB.Bson.Tests/Serialization/SerializationTests.cs

```diff
@@ -14,30 +14,44 @@
 */
 
 using System;
+using System.Linq;
 using System.Reflection;
 using FluentAssertions;
 using MongoDB.Bson.Serialization;
 using MongoDB.Bson.Serialization.Attributes;
 using MongoDB.Bson.TestHelpers;
 using Xunit;
 
-namespace MongoDB.Bson.Tests.Jira
+namespace MongoDB.Bson.Tests.Serialization
 {
-    public class CSharp1559Tests
+    public class SerializationTests
```

Comment:
Why is this file renamed? And if we want to rename it we need a better name ("SerializationTests" is too general).

Reply:
I will revert it